test/cppsuite/tests/main.cxx

```diff
@@ -106,20 +91,53 @@ print_error(const std::string &str)
     std::cerr << "No value given for option " << str << std::endl;
 }
 
+/*
+ * Run a specific test.
+ * config_name is the configuration name. The default configuration is used if it is left empty.
+ */
+int64_t
+run_test(const std::string &test_name, const std::string &config_name = "")
+{
+    std::string cfg, cfg_path;
+    int error_code = 0;
+
+    if (config_name.empty())
+        cfg_path = "configs/config_" + test_name + "_default.txt";
+    else
+        cfg_path = config_name;
+    cfg = parse_configuration_from_file(cfg_path);
+
+    std::cout << "Configuration\t:" << cfg << std::endl;
+
+    if (test_name == "poc_test")
+        poc_test(cfg, test_name).run();
+    else {
+        std::cout << "Test not found: " << test_name << std::endl;
```

Comment:
Use the debug_utilprint function for all prints please. Additionally can we change it from std::cout to std::cerr when the log level is ERROR?